js/about/aboutActions.js

```diff
@@ -247,6 +247,32 @@ const AboutActions = {
    */
   editAutofillCreditCard: function (card) {
     ipc.sendToHost(messages.AUTOFILL_SET_CREDIT_CARD, card.toJS(), card.toJS())
+  },
+
+  /**
+   * Dispatches an event to the browser process to register or deregister a datafile
+   *
+   * @param {uuid} The unique ID of the adblock datafile
+   * @param {enable} true if the adBlock data file should be used
+   */
+  updateAdblockDataFiles: function (uuid, enable) {
+    aboutActions.dispatchAction({
+      actionType: appConstants.APP_UPDATE_ADBLOCK_DATAFILES,
+      uuid,
+      enable
+    })
+  },
+
+  /**
+   * Dispatches an event to the renderer process to register or deregister a datafile
+   *
+   * @param {rules} ABP filter syntax rule string
```

Comment:
might be a copy/paste thing, but are these comments right? to register or deregister? (looks like it's updating existing rules that are in place)